gehomesdk/erd/erd_configuration.py

```diff
@@ -203,4 +203,9 @@ def erd_encode(self, value: Any) -> str:
     ErdConfigurationEntry(ErdCode.SAC_SLEEP_MODE, ErdOnOffConverter(), ErdCodeClass.AC_SENSOR),
     ErdConfigurationEntry(ErdCode.SAC_TARGET_TEMPERATURE_RANGE, ErdSacTargetTemperatureRangeConverter(), ErdCodeClass.AC_SENSOR),
     ErdConfigurationEntry(ErdCode.SAC_AUTO_SWING_MODE, ErdOnOffConverter(), ErdCodeClass.AC_SENSOR),
+
+    #Hood
+    ErdConfigurationEntry(ErdCode.HOOD_LIGHT_MODE, ErdHoodLightSettingConverter(), ErdCodeClass.HOOD_SENSOR),
```

Comment:
Should we add a light code class?  I'm guessing we wouldn't want just a generic hood icon for this, we'd want a light bulb or something like that.  I've been using code class to allow us to more easily set device class/icons in HA.

Reply:
It could... There are a few things like this - timer is another one as well. My hood is also using an ERD labeled "OVEN_TIMER"... I'm not worried yet about this - opportunity as this evolves is what I'm thinking.